src/ensembl/production/metadata/updater/core.py

```diff
@@ -333,6 +333,7 @@ def get_assembly_sequences(self, species_id, assembly):
                     accession_info[seq_region_name]["karyotype_rank"] = karyotype_rank
             # Now, create AssemblySequence objects for each unique accession.
             for accession, info in accession_info.items():
+                seq_region_name = accession
```

Comment:
Sorry I don't get the whole logic here: 
- seq_region_name <= accession
- then accession is updated with the first matching names in synonym ?
- `name = accession` is set but not used afterwards  
you'll need to explain this to me, thanks!